CHANGELOG.md

```diff
@@ -5,7 +5,6 @@
   * Limited support for formatting
   * Error squiggles misaligned for expressions near the start of a new line
   * Emmet based abbreviation expansion is not yet supported (See note in readme for how to enable through your settings.json)
-  * Directive and TagHelper Attribute Completions are now committed contextually with `=`, `:` and ` ` commit characters. However, there's a known issue by which these characters will be added to the attribute value (`@att|` commit using `=` yields `@attributes="=|"` instead of `@attributes="|"` as intended). ([#22150](https://github.com/dotnet/aspnetcore/issues/22150))
```

Comment:
Shouldn't this be:
- Directive and TagHelper Attribute Completions are now committed contextually with `=`, `:` and ` ` commit characters.
- Removed TagHelper attribute completion snippet support as we felt the typing experience was superior without it.

Reply:
Thanks, added that to the new features https://github.com/OmniSharp/omnisharp-vscode/pull/3808/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR25-R26